refresh/templates/common/Package.swift

```diff
@@ -11,6 +11,7 @@ let package = Package(
     dependencies: [
         .Package(url: "https://github.com/IBM-Swift/Kitura.git",             majorVersion: 1, minor: 7),
         .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git",       majorVersion: 1, minor: 7),
+        .Package(url: "https://github.com/IBM-Swift/Health.git",             majorVersion: 0),
```

Comment:
@tunniclm We (swift tooling team) own the Health package; we can make changes to it as needed. We will bump up the major number if we make an API breaking change. We would like to get this code included in the swift generator as soon as possible.